File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    let str = ''
    if(s.length % 2 === 0){
        str = s[(s.length) / 2 - 1] + s[(s.length) / 2]
    }
    else {
        str = s[(s.length-1) / 2]       
    }
    return str
}

File: 프로그래머스/lv1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String str = null;
        String[] sArr = s.split("");
        return s.length() % 2 == 0 ? sArr[(s.length() / 2) - 1]+ sArr[s.length() / 2] : sArr[(s.length() -1) / 2] ;
    }
}